sql/core/src/test/java/test/org/apache/spark/sql/JavaUDFSuite.java

```diff
@@ -136,6 +138,15 @@ public void udf7Test() {
     }
   }
 
+  @Test
+  public void udf8Test() {
+      spark.udf().register(
```

Comment:
Please, adjust indentation to 2 spaces.

Reply:
I changed to 2 and I re-ran the test, sorry about that!